### PR TITLE
make Team foreign key of Signup (#19)

### PR DESCRIPTION
**Pull request by PetrDlouhy**
addresses issue #19 
based on PR #47

### CODE REVIEW

**Comment by just-paja** [comment]
@PetrDlouhy, failing tests.

**Comment by PetrDlouhy** [comment]
This now comes with the tests.

File: api/admin.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 """Site administration module."""
 from django.contrib import admin
-from .models import Accomodation, AccomodationPhoto, Food, FoodPhoto, \
-    Lector, LectorPhoto, Payment, Participant, Workshop, WorkshopPhoto, Year
+
+from . import models
```
Comment (just-paja) [inline api/admin.py]: This change is not related to the PR. Please make file them separately next time.

File: api/models/participant.py [hunk 2]
```diff
@@ -2,10 +2,12 @@
 from django.conf import settings
 from django.core import mail
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
```
Comment (just-paja) [inline api/models/participant.py]: Calling a function "_" is the most horryfing thing anyone has ever done. I realize it has now become some kind of convention, but I am just not strong enough to approve this without you saying there is no other way to do this.

File: api/models/team.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+"""Import Django models."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
```
Comment (just-paja) [inline api/models/team.py]: Underscore gettext here as well. See what underscore means in other languages as well:
* [Javascript](https://www.npmjs.com/package/underscore)
* [Java](https://github.com/javadev/underscore-java)
* [Perl](http://vti.github.io/underscore-perl/)

...

File: api/models/participant.py (hunk 2 above)
Comment (PetrDlouhy) [inline api/models/participant.py]: This is wide spread convention in Django approved by the documentation: https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#standard-translation

Comment (just-paja) [inline api/models/participant.py]: But that's ridiculous. It's saying "Python's convention is A", "We've decided to ignore A and use A for something else so everybody who does Python will think that A means A but it means B".

The documentation states that the purpose is to: ta-ta-taaaa - - - **Save Typing** This convention is stupid. Saving typing is equivalent for obscuring the code. They wouldn't need to save typing if they did translation where it belongs - on the frontend.

When I look at "_", I see invalid variable name.

Comment (PetrDlouhy) [inline api/models/participant.py]: This usage is encouraged even in Python documentation: https://docs.python.org/3/library/gettext.htm

And I can't find any parameter, that You could feed to `makemessages` command to take else than `_()` as marked text when creating `.po` file.

File: api/models/team.py (hunk 3 above)
Comment (PetrDlouhy) [inline api/models/team.py]: That functionality is mostly embedded in Python itself. Underscore is indeed used as convention for translations in Python.
